### PR TITLE
Run unittests in docker container

### PR DESCRIPTION
**Pull request by avishayp**
### why
I was not able to run `tox` tests on mac. I assume this is fixable, but running tests in a container gives a more predictable environment.

### what

 * Added `test` service to `docker-compose.yml` that runs the unittests.
 * Added note in `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,18 @@ A [Kenneth Reitz](http://kennethreitz.org/bitcoin) Project.
 
 Run locally:
 ```sh
-docker pull kennethreitz/httpbin
```
Comment (javabrett) [inline README.md]: Why is this deleted?  It could result in a stale/cached image.

**Comment by avishayp** [comment]
Closing as it solves my problems but not necessarily useful for anyone else.
